install.sh

```diff
@@ -178,7 +178,7 @@ nvm_check_global_modules() {
     wc -l | tr -d ' '                           # Count entries
   )"
 
-  if [ $MODULE_COUNT -ne 0 ]; then
+  if [ "$MODULE_COUNT" -ne 0 ]; then
```

Comment:
quotes are inappropriate when using `-ne` since shells interpret this as an integer.